LayoutTests/http/tests/site-isolation/mouse-events/iframes-with-tooltips.html

```diff
@@ -0,0 +1,81 @@
+<!-- webkit-test-runner [ SiteIsolationEnabled=true ] -->
+<script src="/js-test-resources/js-test.js"></script>
+<script src="/js-test-resources/ui-helper.js"></script>
+<script>
+description("Verifies that tooltips from iframes are correct.");
+jsTestIsAsync = true;
+
+async function runTest() {
+
+    if (window.testRunner) {
+        {
+            let promise = testRunner.tooltipDidChange();
+            await eventSender.asyncMouseMoveTo(30, 30);
+            result = await promise;
+            result = await promise;
+            shouldBe("result", "'Test Main Frame'");
+        }
+        {
+            let promiseClearing = testRunner.tooltipDidChange();
+            eventSender.asyncMouseMoveTo(100, 100);
+            result = await promiseClearing;
+            shouldBe("result", "''");
+
+            let promise = testRunner.tooltipDidChange();
+            result = await promise;
+            shouldBe("result", "'Tooltip A'");
+        }
+        {
+            let promiseClearing = testRunner.tooltipDidChange();
+            eventSender.asyncMouseMoveTo(300, 300);
+            result = await promiseClearing;
+            shouldBe("result", "''");
+
+            let promise = testRunner.tooltipDidChange();
+            await eventSender.asyncMouseMoveTo(300, 300);
```

Comment:
this second mouse move seems wrong? the clear + set pair come inside a single mousemove, don't they?

Reply:
Yeah, it was late and I didn't check it in the morning...